tachyon/math/polynomials/univariate/univariate_dense_coefficients.h

```diff
@@ -69,8 +71,11 @@ class UnivariateDenseCoefficients {
   }
 
   constexpr static UnivariateDenseCoefficients Random(size_t degree) {
-    return UnivariateDenseCoefficients(
-        base::CreateVector(degree + 1, []() { return F::Random(); }));
+    auto v = base::CreateVector(degree + 1, []() { return F::Random(); });
```

Comment:
Could you explicitly name the type as `std::vector<F>`?
```suggestion
    std::vector<F> v = base::CreateVector(degree + 1, []() { return F::Random(); });